### DISCUSSION TITLE
How to generate Sell/Buy signal from SuperTrendIndicator indecator ?

### DISCUSSION THREAD

**Discussion by akashcse** [ta4j/ta4j#1175] [Ideas]
how to generate Sell/Buy signal using SuperTrendUpperBandIndicator and SuperTrendLowerBandIndicator values

**Comment by TheCookieLab** [comment]
That's going to be based on 1) your interpretation of what the super trend
indicators say about the market, and 2) your particular trading strategy.
For example if you are trading mean reversion then the close price crossing
above the super trend upper band could mean prices are due for correction
so you generate a sell signal. Inversely if the close price crossed the
super trend lower band value then the market could be due for a bounce and
thus you generate a buy signal. Conversely if you're a momentum trader you
could consider the price crossing above the upper band to be bullish and
thus a buy signal. It all depends on your strategy.
